package.json

```diff
@@ -55,7 +55,7 @@
 		"prestart": "npm run -s install-if-deps-outdated",
 		"start": "cross-env WC_ADMIN_PHASE=development npm run build:packages && cross-env WC_ADMIN_PHASE=development npm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"node ./bin/packages/watch.js\"",
 		"pretest": "npm run -s install-if-no-packages",
-		"test": "wp-scripts test-unit-js --config tests/js/jest.config.json",
+		"test": "wp-scripts test-unit-js --config tests/js/jest.config.json --maxWorkers=50%",
```

Comment:
The most crucial change is throttling workers (not fixed to a number so that it should work in more environments)